naff/models/discord/channel.py

```diff
@@ -2218,7 +2217,7 @@ async def create_post(
         self,
         name: str,
         content: str | None,
-        applied_tags: Optional[List[Union["Snowflake_Type", "ThreadTag"]]] = MISSING,
+        applied_tags: Optional[List[Union["Snowflake_Type", "ThreadTag", str]]] = MISSING,
```

Comment:
Should allow a single tag to be passed without putting it in a list

Reply:
Hard disagree, the attribute is `applied_tags`, plural. Unlike components, id rather enforce passing the correct data structure for tags - plus its **really** not a big ask to wrap your payload in a list before using it